app/services/moderator/banish_user.rb

```diff
@@ -1,17 +1,18 @@
 module Moderator
   class BanishUser < ManageActivityAndRoles
+    def self.call(admin:, user:)
+      new(user: user, admin: admin).banish
+    end
+
     attr_reader :user, :admin
 
     def initialize(admin:, user:)
       @user = user
       @admin = admin
     end
 
-    def self.call_banish(admin:, user:)
-      new(user: user, admin: admin).banish
-    end
-
     def banish
+      BanishedUser.create(username: user.username)
```

Comment:
In the future, as we grow, we might also want to track who banished the user, I love that we have that option with this new model

Reply:
@mstruve This seems so likely that I added it now. I did make it optional though, since maybe we'll add a script or something in the future that can bulk banish users and we may not want to create a user account only for that purpose.